docs/concepts/xmtp-versions.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_label: XMTP versions
+sidebar_position: 1
+description: "Learn about the fundamental characteristics of XMTP versions"
+---
+
+# Understand XMTP versions
+
+XMTP currently has three versions: V3, V2, and V1.
+
+### XMTP V3
+
+XMTP V3 refers to the latest version of XMTP which has the following fundamental characteristics:
+
+- Supports group chat. To learn more, see [Build group chat with MLS and XMTP](/docs/build/group-chat.md).
+
```

Comment:
Is the whitespace between bullets deliberate? Just noticed it looks a little awkward on the HTML preview